common/djangoapps/student/views.py

```diff
@@ -349,7 +350,7 @@ def _cert_info(user, course_overview, cert_status, course_mode):  # pylint: disa
         'can_unenroll': status not in DISABLE_UNENROLL_CERT_STATES,
     }
 
-    if (status in ('generating', 'ready', 'notpassing', 'restricted', 'auditing') and
+    if (status in ('generating', 'ready', 'notpassing', 'restricted', 'auditing', 'unverified') and
```

Comment:
@rlucioni Did you see my comment below in this PR about the `CertificateStatuses`? It's not as easy as you would think. I think there are better opportunities to address these in the future. Let me know if you feel strong about this.